src/app.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Form from './form.js'
+import {appState} from './app-state.js'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = appState
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+  handleSubmit(event) {
+    event.preventDefault()
+    const newCardData = new FormData(event.target)
+    const newCard = {
+      question: newCardData.get('question'),
+      answer: newCardData.get('answer')
+    }
+    const addCard = this.state.cards.map(card => Object.assign({}, card))
```

Comment:
When this method is defined in your form component, it'll do something like `this.props.addCard(flashCard)`. The `App` just wants the `flashCard` object when it's ready.